common/tools/mcp-server/README.md

```diff
@@ -0,0 +1,65 @@
+# Azure SDK for JS MCP Server
+
+A Model Context Protocol (MCP) server implementation for the Azure SDK repository. This server is designed to facilitate the development and testing of tools that communicate using the MCP.
+
+> **Note**: This project is in early development stages and may not be fully functional. It is not enabled by default in the Azure SDK for JavaScript repository. As more tools are developed, the server will be enabled for testing.
+
+## Getting Started
+
+### Prerequisites
+
+- [LTS versions of Node.js](https://github.com/nodejs/release#release-schedule)
+
+### Activation
+
+The MCP server can be enabled by adding the following entry under the root ./.vscode/ folder in a file called `mcp.json`:
+
+```json
+{
+  "servers": {
+    "Azure SDK": {
+      "type": "stdio",
+      "command": "npx",
+      "args": ["tsx", "${workspaceFolder}/common/tools/mcp-server/src/index.ts"],
```

Comment:
Should we add this to the config in our repo? Or do we want to kick the tires a bit first? OK either way

Reply:
My thinking was that we leave it off by default until we have some useful tools to add - this will also give us some time to kick the tires like you said. Even one useful tool can be the reason to enable it for everyone